react-frontend/src/css/nav.css

```diff
@@ -17,7 +17,23 @@ header {
     margin: auto;
     width: 1115px;
   }
-  
+
+  .covidBanner {
+    background-color: #fcba19;
+    top: 0;
+    position: fixed;
+    height: 6%;
+    width: 100%;
+    padding: 0 0 0 200px;
+    z-index: 2000;
+    margin: auto;
+  }
+
+  .covidBanner Button {
```

Comment:
For the button that dismisses the alert, gov.bc.ca, instead of using `<button>` uses `<a class="close alert-notification-close">`.

```
.close {
  float: right;
  font-size: 21px;
  font-weight: 700;
  line-height: 1;
  color: #000;
  text-shadow: 0 1px 0 #fff;
  filter: alpha(opacity=20);
  opacity: .2;
}

.alert-notification-close {
  color: #fff !important;
  opacity: 0.9 !important;
  top: -4px !important;
  font-size: xx-large !important;
}

```